src/app/components/profile-location/profile-location.component.html

```diff
@@ -9,8 +9,8 @@ <h4>Location:</h4>
                     <label for="address" class="col-sm-4 col-form-label" >Street Address:</label>
                     <div class="col-sm-4">
                         <input type="text" formControlName="address2" class="form-control" id="address">
-                        <p *ngIf="addressChange.controls.streetAddress2.hasError('required') && addressChange.controls.streetAddress2.touched" class="error">Street Address required please enter one.</p>
-                        <p *ngIf="addressChange.controls.streetAddress2.hasError('pattern')" class="error">Street Address format is incorrect</p>
+                        <p *ngIf="addressChange.controls.address2.hasError('required') && addressChange.controls.address2.touched" class="error">Street Address required please enter one.</p>
```

Comment:
Is this change needed in order for the profile-location component to work?

Reply:
Yes, this change is needed because streetAddress2 is the wrong variable name, so without that change it will say something along the lines of cannot read property hasError of undefined.

Reply:
Alrighty, that should be fine then.